bin/hdfcoinc/pycbc_page_snrifar

```diff
@@ -81,8 +151,17 @@ except:
 if cstat_fore is not None and len(cstat_fore) == 0:
     cstat_fore = None
 
-back_ifar = f['background/ifar'][:]
-cstat_back = f['background/stat'][:]
+# Make choosing the background backwards compatible
```

Comment:
New code added to avoid breaking when no hierarchical removals are done starts here.